mpifileutils.spec

```
@@ -43,15 +43,15 @@
 %endif
 
 Name:		mpifileutils
-Version:	0.10.1
-Release:	6%{?relval}%{?dist}
+Version:	0.11
+Release:	7%{?relval}%{?dist}
```

When the `Version:` is incremented, the `Release:` goes back to `1`.
```suggestion
Release:	1%{?relval}%{?dist}
```

---

I think I will also need to update daos-stack/daos#4511 and daos-stack/lwgrp#3 in the same manner.